app-backend/tile/src/main/scala/directives/TileAuthentication.scala

```diff
@@ -39,8 +39,8 @@ trait TileAuthentication extends Authentication
   }
 
   def isProjectPublic(id: UUID): Directive1[Boolean] =
-    onSuccess(Projects.getProject(id)).flatMap {
-      case Some(project) => provide(project.tileVisibility == Visibility.Public)
+    onSuccess(Projects.getPublicProject(id)).flatMap {
+      case Some(_) => provide(true)
```

Comment:
I just tried this but am getting this error:

```
[error] /Users/ttuhinanshu/dev/rf/app-backend/tile/src/main/scala/directives/TileAuthentication.scala:42: missing parameter type for expanded function ((x$1) => x$1.isDefined)
[error]     onSuccess(Projects.getPublicProject(id)).flatMap(provide(_.isDefined))
[error]                                                              ^
```

I'm actually fine with keeping it as is. It's more readable and aligns better with other similar responses.